### PR TITLE
created basic layout of the Create Users page

### PR DESCRIPTION
**Pull request by gdioso**
- Created the "Users" page
- Added a link for the "delete" icon in index.html
- Installed @headlessui/react for creating dialogs
- Added a "UsersDialog" component

The "UsersDialog" component is mostly done. All that is left is adding functionality for adding new users.

### CODE REVIEW

**Comment by Sillor** [comment]
Please pull from `main` first and resolve the merge conflict.

**Comment by AlissonRoss** [comment]
image
Nice

**Review by AlissonRoss — DISMISSED** [review]

**Review by lclemente225 — APPROVED** [review]
Looks good

File: src/App.jsx
Comment (lclemente225) [inline src/App.jsx:1]: Note: This is the merge conflict. Not a big issue.

**Review by nlawler1737 — DISMISSED** [review]
Looks Good!

**Comment by AlissonRoss** [comment]
Looking good on my end :D 
image

**Review by AlissonRoss — DISMISSED** [review]

**Review by nlawler1737 — APPROVED** [review]
loods good!

**Review by nlawler1737 — APPROVED** [review]
looks good

**Review by AlissonRoss — APPROVED** [review]
Looking good on my end

**Pull request merged**